shapely/tests/test_constructive.py

```diff
@@ -972,6 +972,34 @@ def test_minimum_bounding_circle(geometry, expected):
     assert_geometries_equal(actual, expected)
 
 
+@pytest.mark.parametrize("geometry", all_types)
+def test_minimum_width_all_types(geometry):
+    actual = shapely.minimum_width([geometry, geometry])
```

Comment:
You could also add `shapely.minimum_width` to `CONSTRUCTIVE_NO_ARGS` defined above and then this test should be covered by an existing one